README.md

```diff
@@ -1,16 +1,230 @@
-# Title
+# JavaScript Fundamentals: Logical Operators
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+* Use `!` to negate an expression
+* Convert an expression to a boolean using `!!`
+* Link conditions with the `&&` and `||` operators
 
 ## Introduction
 
-## SWBAT 1
+In this lesson, we'll learn how to negate and combine expressions with
```

Comment:
This is just wording, but I'm not a huge fan of the "in this lesson" intro. If it's possible to find a more organic way to begin, that might be better. On the other hand, this is a pretty straightforward topic, so it's sometimes harder to be creative with those.